src/main/java/com/conveyal/datatools/manager/controllers/api/UserController.java

```diff
@@ -167,24 +182,56 @@ private static String createPublicUser(Request req, Response res) {
     }
 
     /**
-     * HTTP endpoint to create new Auth0 user for the application.
+     * HTTP endpoint to create new Auth0 user for the application. If the Auth0 user already exists, update the user
+     * permissions instead.
      */
     private static String createUser(Request req, Response res) {
+        JsonNode jsonNode = parseJsonFromBody(req);
+        String email = jsonNode.get("email").asText();
+        Auth0UserProfile user = checkForExistingAuth0User(email);
+
+        // User already exists, update permissions
+        if (user != null) {
+            String json = String.format("{ \"app_metadata\": { \"datatools\" : %s  }}", jsonNode.get("permissions"));
+            return updateUserPermissions(req, user, json);
```

Comment:
I think the datatools key should have an array value (see below at line 206).

Reply:
Yep, had spotted that, will be in the next commit.

Reply:
Updated.

Reply:
Turns out that the array value [] is provided so isn't needed here. I have updated with a comment.

Reply:
Updating permissions causes loss of existing permissions that were in place for that Auth0 user, so this won't work if an agency manages multiple instances (e.g. test and production) from the same Auth0 connection.
I think we want to append the permissions if it is an existing auth0 user.

Reply:
Also, I pulled the latest code and the `datatools` brackets seem to be still not inserted:
![image](https://user-images.githubusercontent.com/56846598/94310286-65913f00-ff47-11ea-8862-e325921c3bf8.png)